avb/libavb_ab/avb_ab_flow.c

```diff
@@ -61,7 +61,7 @@ void avb_ab_data_init(AvbABData* data) {
   avb_memset(data, '\0', sizeof(AvbABData));
   data->magic = BOOT_CTRL_MAGIC;
   data->version_major = BOOT_CTRL_VERSION;
-  data->nb_slot = sizeof(data->slot_info)/sizeof(data->slot_info[0]);
```

Comment:
so, do we need to support the case that a/b is disabled? is there any switchers in mixin to enable/disable a/b?

Reply:
No. AB is must for AVB. There is no switcher to control it. The code is within ab implement directory. All slot number is hardcoded to 2.